src/cmds.js

```diff
@@ -127,16 +127,75 @@ const getRemoteUrls = ({ commitPath, comparePath, prPath, remoteUrl } = OPTIONS)
  *
  * @param {object} settings
  * @param {Function} settings.getReleaseCommit
- * @returns {string}
+ * @param {Array} settings.breakingChangeMessageFilter
+ * @param {Array} settings.breakingChangeScopeTypeFilter
+ * @returns {Array}
  */
-const getGit = ({ getReleaseCommit: getAliasReleaseCommit = getReleaseCommit } = {}) => {
+const getGit = ({
+  getReleaseCommit: getAliasReleaseCommit = getReleaseCommit,
+  breakingChangeMessageFilter = ['BREAKING CHANGE:', 'BREAKING CHANGES:'],
+  breakingChangeScopeTypeFilter = [')!:', '!:']
+} = {}) => {
   const releaseCommitHash = getAliasReleaseCommit().split(/\s/)[0];
+  let breakingChangeMessageCommits;
+  let breakingChangeScopeTypeCommits;
+  let commits;
 
-  if (!releaseCommitHash) {
-    return runCmd(`git log --pretty=oneline`, 'Skipping commit "get" check... {0}');
-  }
+  /**
+   * Build git log command
+   *
+   * @param {string} commitHash
+   * @param {Array} searchFilter
+   * @returns {string}
+   */
+  const getGitLog = (commitHash, searchFilter) => {
+    const releaseCommitHashRange = (commitHash && ` ${commitHash}..HEAD`) || '';
+    const searchFilterCommits = searchFilter?.map(value => ` --grep="${value}"`).join(' ') || '';
+
+    return runCmd(
+      `git log${releaseCommitHashRange} --pretty=oneline${searchFilterCommits}`,
+      'Skipping commit "get" check... {0}'
+    );
+  };
+
+  breakingChangeMessageCommits = getGitLog(releaseCommitHash, breakingChangeMessageFilter);
+  breakingChangeScopeTypeCommits = getGitLog(releaseCommitHash, breakingChangeScopeTypeFilter);
+  commits = getGitLog(releaseCommitHash);
+
+  breakingChangeMessageCommits = breakingChangeMessageCommits
+    .trim()
+    .split(/\n/g)
+    ?.filter(value => value !== '');
+
+  breakingChangeScopeTypeCommits = breakingChangeScopeTypeCommits
```

Comment:
required: 
- put a note here, this logic needs to be explained around why we have to double check the git grep
- rethink returning `true` as the fallback, originally we did this when it was combined with the `breakingChangeMessageCommits` git grep because we couldn't necessarily determine which "grep"/lookup/filter was doing what